db/models.py

```diff
@@ -1 +1,16 @@
 from django.db import models
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=255)
+
+    def __str__(self) -> str:
+        return f"Genre: {self.name}"
```

Comment:
You don't need to indicate the model name, it is already added in a final queryest and may lead to name duplication. The same relates to the `Actor` class as well.

Reply:
This comment is not implemented, return just `self.name` for `__str__` for `Gengre` class and preserve the type annotation with return type `str`. Return just `f"{self.first_name} {self.last_name}"` for `__str__` for `Actor` class.